### PR TITLE
test: Create unit tests for react components in packages/ui/components/head-seo/HeadSeo

### PR DESCRIPTION
**Pull request by gitstart-calcom**
```
yarn test head-seo
```
image

### CODE REVIEW

File: packages/ui/components/head-seo/head-seo.test.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      const titleEl = container.querySelector("title");
+      expect(titleEl?.getAttribute("canonical")).toEqual(basicProps.canonical);
+      expect(titleEl?.getAttribute("description")).toEqual(basicProps.description);
+      expect(titleEl?.getAttribute("site_name")).toEqual(basicProps.siteName);
+      expect(titleEl?.getAttribute("title")).toEqual(`${basicProps.title} | Cal.com`);
+      expect(titleEl?.getAttribute("image")).toContain("constructGenericImage");
+    });
+  });
+
+  test("Should render without brand", async () => {
```
Comment (hariombalhara) [inline packages/ui/components/head-seo/head-seo.test.tsx:60]: We need a complementary test as well that verifies that it rendered with brand

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left 1 comment. Thanks @gitstart-calcom

File: packages/ui/components/head-seo/head-seo.test.tsx (hunk 1 above)
Comment (gitstart-calcom) [reply packages/ui/components/head-seo/head-seo.test.tsx:60]: @hariombalhara
Rendering with branding is the default behavior, this is being tested on line 55

Comment (hariombalhara) [reply packages/ui/components/head-seo/head-seo.test.tsx:60]: Oh cool !! I think it would make sense to extract that out in a separate test.

Comment (gitstart-calcom) [reply packages/ui/components/head-seo/head-seo.test.tsx:60]: Sure, we will make the required changes and get back to you

**Comment by gitstart-calcom** [comment]
Hi @hariombalhara we made the requested changes please take a look 😄

**Review by hariombalhara — APPROVED** [review]
LGTM!!

**Pull request merged**